install-dev/langs/en/data/tab.xml

```diff
@@ -129,5 +129,5 @@
     <tab id="Warehouses" name="Warehouses" />
     <tab id="Webservice" name="Webservice" />
     <tab id="Zones" name="Zones" />
-    <tab id="Marketing_tools" name="Modules Selections" />
+    <tab id="Marketing_tools" name="Module Selections" />
```

Comment:
I already asked about that string. The previous one was correct.

Reply:
cf. the famous "s" question about the feature's title. ;-)

Reply:
No S no S :D